docs/accessibility.md

```diff
@@ -0,0 +1,221 @@
+# Accessibility
+
+## General Practices
+
+As a general rule, accessible software is achieved by following web development best practices. As Treeherder uses the `reactstrap` library, it is recommended using one of its components, if possible.
+
+If you can not find a suitable component, prefer [HTML5 semantic tags](https://developer.mozilla.org/en-US/docs/Glossary/Semantics#Semantics_in_HTML) and avoid using generic tags, such as `span` and `div`. Moreover, try to use more **headings** (`h1` to `h6`) whenever possible, because screen readers users usually navigate by headings and that is only possible when developers insert them in the code.
+
+If you can build your layout neither with `reactstrap` nor with semantic tags, please insert `aria-role` in them. There are examples of how to use `aria-role` in: [Tables](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Table_Role), [Progress Bar](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_progressbar_role), [List](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/List_role).
+
+`aria-role` is important because it gives meaning to the generic HTML tag. That gives some direction to screen readers on which functionalities to provide and some information to users.
+
+**In summary**, implement your layout trying to use these in the following order:
+
+1. Use `reactstrap`;
+2. Use HTML semantic tags;
+3. Use `aria-role` attributes.
+
+### Reactstrap
+
+Some elements in Treeherder's UI are borrowed from the `reactstrap` library. As a general rule, its elements are already accessible.
+That library is built on top of Bootstrap, therefore the [Bootstrap Accessibility documentation](https://getbootstrap.com/docs/4.0/getting-started/accessibility/) is the one to look for, if in doubt.
+
+There are some issues on **Color Contrast** that are pointed by Bootstrap itself. This was solved in Treeherder by providing [**_custom color classes_**](accessibility.md#colors).
+
+### Font Awesome
+
+Treeherder also uses the `Font Awesome` library for some icons. Usually, there is no need to insert an additional explanation to them. However, if the icon has a semantic meaning, you can insert it by adding a `title` prop in the `FontAwesomeIcon` component.
+
+```jsx
+<FontAwesomeIcon icon={icon} title="Icon description" />
+```
+
+## Colors
+
+Treeherder uses [Bootstrap color](https://getbootstrap.com/docs/4.0/utilities/colors/) utility classes. While most of them work, there are a couple that needed a little tweaking to increase **color contrast**, according to [WCAG standards](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast?utm_source=devtools&utm_medium=a11y-panel-checks-color-contrast).
+
+So, we ask that you substitute the Bootstrap color to the Treeherder one, according to the following table:
```

Comment:
"Please use this custom color according to the following table:"

Reply:
This can be a follow up, but did you make changes to red and green or was it only for specific Treeherder buttons on the Job View UI (I can't remember)?

Reply:
>    This can be a follow up, but did you make changes to red and green or was it only for specific Treeherder buttons on the Job View UI (I can't remember)?

I did change their colors, but the reds and greens and the other colors on the Jobs View were not from Bootstrap, they were custom made. They are in `treeherder-job-buttons.css`.